### ISSUE TITLE
[Request] Integration reconfiguring

### ISSUE THREAD

**Issue by KairuByte** [LordBoos/discord_game#56]
It would be nice to be able to reconfigure the integration without having to remove and re-add it. Even just re-running the original wizard with existing values pre-filled would be a massive improvement.

**Comment by LordBoos** [comment]
Yes, it is planned, but a bit complicated right now. In the meantime, if you want to add new users, you don't need to remove the integration, you can just add another one with only those new users. You will have 2 entries in the services, but it will work without any issues. I use it like that myself right now. 
![image](https://github.com/LordBoos/discord_game/assets/939813/f2d4a77b-63dd-4080-bdf2-c97188e3279e)

**Comment by Masteroshi430** [comment]
Hi, I updated to 6.0.3 and the integration lost the access_token, I've tried to uninstall/reinstall the integration but I don't have access to configuration anymore to re-enter it.
